portal-api/api-tests/v1/gef/external-api.api-test.js

```diff
@@ -13,7 +13,6 @@ xdescribe(baseUrl, () => {
   });
 
   beforeEach(async () => {
```

Comment:
We can remove this completely if it's empty

Reply:
Done, "beforeEach" is removed